package.json

```diff
@@ -12,9 +12,11 @@
   "dependencies": {
     "@faker-js/faker": "^9.0.0",
     "bootstrap": "^5.3.3",
-    "popper.js": "^1.16.1",
     "husky": "^9.1.6",
+    "popper.js": "^1.16.1",
```

Comment:
Using this?

Reply:
no its for positioning, ill remove for now